src/debug_utils.cc

```diff
@@ -27,10 +27,12 @@
 #endif  // __POSIX__
 
 #if defined(__linux__) || defined(__sun) || \
-    defined(__FreeBSD__) || defined(__OpenBSD__)
+    defined(__FreeBSD__) || defined(__OpenBSD__) || \
+    defined(__DragonFly__)
 #include <link.h>
 #endif  // (__linux__) || defined(__sun) ||
```

Comment:
While we're here, can we fix these comments? I'm not sure why we have the 'defined' part for only two of the platforms?

Reply:
Also, if the `#if` “body” is only one or two lines, I’d omit the comment on the closing `#endif` altogether.